subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/resolveengine/graph/builder/ComponentState.java

```diff
@@ -290,7 +292,7 @@ public boolean isFromPendingNode() {
         return selectedBy != null && selectedBy.getDependencyMetadata().isPending();
     }
 
-    boolean isSelected() {
+    public boolean isSelected() {
```

Comment:
Looks like this is leftover from the previous spike.

Reply:
Nope, it's required in conflict resolution.